src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -103,8 +108,9 @@ class Task extends Component {
             <div className="task-assigned-to">
               <span>
                 {
-                  (assignedPursuanceId && pursuances[assignedPursuanceId].suggestionName)
-                  ||
+                  assignedPursuanceId && pursuances[assignedPursuanceId].suggestionName
+                }
+                {
```

Comment:
@Moe-Shoman This logic used to show the name of the assigned pursuance || the name of the assigned user. With this change, if a pursuance and a user are both set as the assignee of the task, this will show both.

I'll think about it, but I'm pretty sure we want: if `task.assigned_to_pursuance_id` is set, and if it's != `Number(this.props.pursuanceId)`, render `pursuances[assignedPursuanceId].suggestionName`, else, render `task.assigned_to && '@' + task.assigned_to`.

Reply:
@elimisteve So a task can be assigned to a pursuance and a user?

Reply:
Sorry, I remember we talked about this but I can't remember the scenario where it would be assigned to both

Reply:
Yes. It could get assigned to another pursuance, then that other pursuance could assign it to a specific user in that pursuance.  So the pursuance that assigned the task to some other pursuance should see the name of the pursuance that was assigned, but the pursuance that was assigned the task should see the particular user who the pursuance is assigned to.

In addition to that, as we were saying in person, somehow indicating that the task came from another pursuance would be good...  We could show the assigning pursuance by saying something like `Assigned by {pursuances[task.pursuance_id].name}`?

Reply:
No problem; it's complicated :-)

Reply:
Right! or maybe have it in a different color or something like that

Reply:
Yeah! We could say "Assigned by ____" in the task details, and we could use another color or use some icon or something in the Task Hierarchy view?

For now, want to do the `Assigned by {pursuances[task.pursuance_id].name}` thing and mess around with colors/maybe do something more complex later after we've thought about it more/talked to a designer/etc?

Reply:
Sounds good, should I revert it back to the way it was or implement this `"if task.assigned_to_pursuance_id is set, and if it's != Number(this.props.pursuanceId), render"`?

Reply:
Implementing that would be great :+1: since tasks are about to show up in both the pursuance requesting for the help and the pursuance being asked for the help on that task.

Reply:
Sounds good

Reply:
or I can do it in the morning, since I'm basically laying out the complex logic in pseudocode; I can do it if you'd prefer.